action.yml

```diff
@@ -26,6 +26,10 @@ inputs:
     description: "IAM session name that will tag your sts-assume-role operation."
     required: false
     default: 'undefined'
```

Comment:
two actions left:

1. please update the readme and explain the new property, remark it is optional
2. Test this GH action, update your workflow to target terraform-apply-gh-action@{YOUR_BRANCH} if it works please  attach the evidence